helm/cluster-operator/templates/service.yaml

```
@@ -1,14 +1,14 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: cluster-operator
-  namespace: {{ .Values.namespace }}
+  name: {{ tpl .Values.resource.default.name  . }}
+  namespace: {{ tpl .Values.resource.default.namespace  . }}
   labels:
-    app: cluster-operator
+    app: {{ tpl .Values.resource.default.name  . }}
```

We should have version label here as well.

---

The `app` label should be `cluster-operator` here also.

```suggestion
    app: {{ .Values.project.name }}
```

---

Done